tests/Validators/StringValidatorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Validators;
+
+use App\Validators\StringValidator;
+use Tests\TestCase;
+
+class StringValidatorTest extends TestCase
+{
+    public function testValidateExistsAndLengthSuccess()
+    {
+        $maxCharacterLength = 30;
+        $name = 'Loads of stocks';
+        $result = StringValidator::validateExistsAndLength($name, $maxCharacterLength);
+        $this->assertEquals('Loads of stocks', $result);
+    }
+
+    public function testValidateExistsAndLengthFailure()
+    {
+        $maxCharacterLength = 30;
+        $name = '';
+        $this->expectException(\Exception::class);  
+        $result = StringValidator::validateExistsAndLength($name, $maxCharacterLength);    
+    }
+
```

Comment:
I think we are also missing a unit test to make sure that if our input is larger than the max characters we set, it also throws an exception.